README.md

```diff
@@ -18,7 +18,10 @@ You can also refer to the following YouTube video for more information on how to
 
 ## What's in This Repository
 
-...
+This repository contains projects that display information in a tree from [self-referential (flat)](https://docs.devexpress.com/MAUI/404899/tree-view/display-hierarchical-data#self-referential-data-structure) and [hierarchical](https://docs.devexpress.com/MAUI/404899/tree-view/display-hierarchical-data#hierarchical-data-structure) data structures:
```

Comment:
I wanted to rewrite that a little, because it's a very quick transition from "tree" to "flat" and I was a bit confused. The first time I read the sentence I actually read it "in a tree FORM", which obviously didn't make much sense... Anyway, the sentence is rather long and I want to have one complete thought first - we display things in a Tree View control. And then we list possible data source types. 

```suggestion
This repository contains projects that populate DevExpress .NET MAUI Tree View control with data. Two types of data sources can supply values to the control: [self-referential (flat)](https://docs.devexpress.com/MAUI/404899/tree-view/display-hierarchical-data#self-referential-data-structure) and [hierarchical](https://docs.devexpress.com/MAUI/404899/tree-view/display-hierarchical-data#hierarchical-data-structure):
```

Reply:
The first sentence looks good, but the second one is misleading a bit. I do not think that we can say **hierarchical data source** or **self-referencial data source**. It is about a data structure within data source. But we can say that the data source contains flat data (self-referencial data) or hierarchical data. In this case, I would extend the first sentence in the following way:  "This repository contains projects that populate DevExpress .NET MAUI Tree View control with self-referential (flat) and hierarchical data."